### PR TITLE
Add inspection tests

### PR DESCRIPTION
**Pull request by yahavi**
Add inspection tests and fix some minor issues

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/inspections/AbstractInspection.java [hunk 1]
```diff
@@ -210,6 +216,54 @@ DependenciesTree getRootDependenciesTree(PsiElement element) {
...
+                .map(DependenciesTree::getGeneralInfo)
+                .filter(Objects::nonNull)
+                .filter(grandChildGeneralInfo -> compareGeneralInfos(generalInfo, grandChildGeneralInfo))
+                .findAny()
+                .ifPresent(grandChildGeneralInfo -> modules.add(child)));
+        return modules;
+    }
+
+    /**
+     * Return true iff the dependency stated in the General info is a module in the project.
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/inspections/AbstractInspection.java]: iff --> if

File: src/main/java/com/jfrog/ide/idea/inspections/AbstractInspection.java [hunk 2]
```diff
@@ -210,6 +216,54 @@ DependenciesTree getRootDependenciesTree(PsiElement element) {
...
+     *
+     * @param node        - The project node
+     * @param generalInfo - General info of the dependency
+     * @return true iff the dependency stated in the General info is a module in the project
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/inspections/AbstractInspection.java]: iff --> if

File: src/main/java/com/jfrog/ide/idea/inspections/AbstractInspection.java (hunk 1 above)
Comment (yahavi) [reply src/main/java/com/jfrog/ide/idea/inspections/AbstractInspection.java]: iff == if and only if

**Comment by yahavi** [comment]
I tried to use the resources dir, but the test framework persists it should be under testData. I tried and It didn't work under resources. Please take look at their documentation under
[CodeInsightTestFixture::configureByFile](https://github.com/JetBrains/intellij-community/blob/master/platform/testFramework/src/com/intellij/testFramework/fixtures/CodeInsightTestFixture.java#L126).

**Pull request merged**